packages/headless/src/controllers/commerce/core/facets/headless-core-commerce-facet.ts

```diff
@@ -281,12 +286,20 @@ export function buildCoreCommerceFacet<
 
     get state() {
       const response = getResponse();
+      const canShowMoreValues = response?.moreValuesAvailable ?? false;
+
+      const canShowLessValues = (() => {
```

Comment:
Weird-ish IIFE here. Can we just extract it in a dedicated named function ?

Reply:
Addressed this in https://github.com/coveo/ui-kit/pull/3495/commits/6dd9a472cc7bad15df5dcaebce244f2e21ae37ea